overrides/main.html

```diff
@@ -0,0 +1,7 @@
+{% extends "base.html" %}
+
+{% block announce %}
+  <div id="announce">
+    <p>We just released 💠 Grid, 🟰 Row, and more extras! Check out more in this <strong><a style="color:white" href="https://twitter.com/streamlit/status/1689742703657918466">announcement tweet</a></strong> from the Streamlit team.</p>
```

Comment:
This is so cool :)

Reply:
Hey what about st.announce() in Streamlit OSS? 🧠